libs/providers/flagd-web/project.json

```diff
@@ -76,7 +74,7 @@
       "executor": "nx:run-commands",
       "options": {
         "commands": [
-          "npx jest"
+          "npx jest --detectOpenHandles --runInBand"
```

Comment:
I think we can remove at least `--detectOpenHandles`? and possibly `--runInBand`? I guess this was from troubleshooting this hanging handles?